extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -167,13 +167,15 @@ attr_lists: {
   attrs: {
     name: "src"
     value_url: {
+      protocol: "http"
```

Comment:
why relaxing this rule?

Reply:
Thanks for mentioning this, I wanted to discuss it. `amp-story` allows http protocol for its configuration JSON blobs, and I didn't understand why we would forbid http for amp-date-picker.

I do see this in the docs so maybe it would be best to keep it.

> Note that in many cases, you may want to allow only "https" as non-secure resources will
generate mixed-mode warnings when displayed from the AMP Cache.
>
> https://github.com/ampproject/amphtml/blob/11f849c/contributing/component-validator-rules.md